operations/business-operations/analytics/metrics-definitions.md

```diff
@@ -338,41 +338,58 @@ Examples include: The customer requests that the ticket remain open after a solu
 #### Ticket Measures
 
 * **First reply time:** The duration between ticket creation and the first public agent reply on the ticket.
-* **Next reply time:** The duration between ticket's first reply time and next reply time, unless otherwise communicated.
+* **Next reply time:** The duration between a customers ticket update and the next public agent reply on the ticket, unless otherwise communicated.
```

Comment:
```suggestion
* **Next reply time:** The duration between a customer's ticket update and the next public agent reply on the ticket, unless otherwise communicated.
```